### PR TITLE
SOV-2445: Create documentation for Atoms in Gitbook (part 1)

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.atlassian.net/browse/SOV-2445

### CODE REVIEW

File: packages/ui/src/1_atoms/Button/Button.stories.tsx [hunk 1]
```diff
@@ -24,6 +24,54 @@ Default.args = {
...
+    description:
+      'The onClick handler for the button, triggered whenever the button is clicked',
+  },
+  text: {
+    control: 'text',
+    description:
+      'The content of the button. Can be text, other components, or HTML elements.',
+  },
+};
+
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/Button/Button.stories.tsx:84]: missing description for `hrefExternal` and `href` props

File: packages/ui/src/1_atoms/Checkbox/Checkbox.stories.tsx [hunk 2]
```diff
@@ -108,6 +108,42 @@ BaseCheckbox.args = {
...
+    description:
+      'The onChangeValue handler for the input, triggered whenever the checkbox value changes',
+  },
+  onChange: {
+    control: 'function',
+    note: '@deprecated Use onChangeValue if possible',
+    description:
+      'The onChangeValue handler for the input, triggered whenever the checkbox value changes',
+  },
+};
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/Checkbox/Checkbox.stories.tsx:150]: missing description for `checked` prop

File: packages/ui/src/1_atoms/ErrorBadge/ErrorBadge.stories.tsx [hunk 3]
```diff
@@ -21,3 +21,23 @@ Default.args = {
...
+  message: {
+    control: 'text',
+    description: 'The content of the badge',
+  },
+  level: {
+    control: 'select',
+    options: Object.values(ErrorLevel),
+    description: 'The style to apply to the badge based on the erorr level',
+  },
+};
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/ErrorBadge/ErrorBadge.stories.tsx:43]: missing description for `weight` prop

File: packages/ui/src/1_atoms/Heading/Heading.stories.tsx [hunk 4]
```diff
@@ -19,6 +19,27 @@ export const Default = Template.bind({});
...
+      'The content of the heading. Can be text, other components, or HTML elements.',
+  },
+  type: {
+    control: 'select',
+    options: Object.values(HeadingType),
+    description: 'The type of the element',
+  },
+};
 
 export const All = () => (
```
Comment (pietro-maximoff) [inline packages/ui/src/1_atoms/Heading/Heading.stories.tsx]: missing description for the details on the `All` page

**Review by pietro-maximoff — CHANGES_REQUESTED** [review]

File: packages/ui/src/1_atoms/ErrorBadge/ErrorBadge.stories.tsx (hunk 3 above)
Comment (Rickk137) [reply packages/ui/src/1_atoms/ErrorBadge/ErrorBadge.stories.tsx:43]: the weight is not actually used for the component. It's used in ErrorsList component to set the priority

**Review by pietro-maximoff — APPROVED** [review]

**Pull request merged**